modules/sievefilters/modules.php

```diff
@@ -635,7 +675,7 @@ public function process() {
         $script_name = generate_filter_name($this->request->post['sieve_filter_name'], $priority);
         $conditions = json_decode($this->request->post['conditions_json']);
         $actions = json_decode($this->request->post['actions_json']);
-        $test_type = mb_strtolower($this->request->post['filter_test_type']);
+        $test_type = strtolower($this->request->post['filter_test_type']);
```

Comment:
Is this intentional?

I am wondering because of https://github.com/cypht-org/cypht/issues/1224

Reply:
no it was not intentional. I understand, I correct